.github/CHANGELOG.md

```diff
@@ -3,13 +3,45 @@
 <h3>New features since last release</h3>
 
 * Adds the new template `AllSinglesDoubles` to prepare quantum states of molecules
-  using the `SingleExcitation` and `DoubleExcitation`. 
+  using the `SingleExcitation` and `DoubleExcitation` operations. 
   The new template reduces significantly the number of operations
   and the depth of the quantum circuit with respect to the traditional UCCSD
-  unitary. The performance of quantum chemistry simulations should benefit
-  from this new implementation. 
+  unitary.
   [(#1383)](https://github.com/PennyLaneAI/pennylane/pull/1383)
 
+  For example, consider the case of two particles and four qubits.
+  First, we define the Hartree-Fock initial state and generate all
+  possible single and double excitations. 
+
+  ```python
+  import pennylane as qml
+  from pennylane import numpy as np
+
+  electrons = 2
+  qubits = 4
+
+  hf_state = qml.qchem.hf_state(electrons, qubits)
+  singles, doubles = qml.qchem.excitations(electrons, qubits)
+  ```
+  Now we can use the template ``AllSinglesDoubles`` to define the
+  quantum circuit,
+
+  ```python
+  wires = range(qubits)
+
+  @qml.qnode(dev)
+  def circuit(weights, hf_state, singles, doubles):
+      qml.templates.AllSinglesDoubles(weights, wires, hf_state, singles, doubles)
```

Comment:
This is really long! I guess there's not much we can do about it, but it's a shame

Reply:
Could import the template and save the `qml.templates` part? I think we often do that in docstrings